_includes/pattern.html

```diff
@@ -50,6 +50,11 @@ <h2 class="alert-title">{{ post.title }} requires the edX UI Toolkit</h2>
                 <li class="pldoc-tab-label">
                     <button class="pldoc-link pldoc-link-markup" data-href="#{{post.slug}}-markup">Markup</button>
                 </li>
+                {% if post.tags contains "js" %}
+                <li class="pldoc-tab-label">
```

Comment:
The rest of the file doesn't follow this convention, and it would mean the ```li``` would be at a different indentation that its siblings so until we standardize on that convention I would like to leave as is for now.